src/us/kbase/workspace/database/ObjectInformation.java

```diff
@@ -239,54 +188,10 @@ private Reference getLast(final List<Reference> refpath) {
 		return refpath.get(refpath.size() - 1);
 	}
 
-	@Override
-	public String toString() {
```

Comment:
`toString()` is too much of a pain to test given how easy it is to just add it in temporarily when you need it.